docs/example-scenario/data/hybrid-etl-with-adf.md

```diff
@@ -0,0 +1,102 @@
+Server potential on-premise use cases are listed below:
+
+* SSIS package used to load network router logs to a database for analysis
+* SSIS package used to prepare and clean human resource employment data used for analytics reporting
+* SSIS package used to load product and sales data into a data warehouse for sales forecasting
+* SSIS package used to automate ODS/DW loading for finance and accounting
+
+## Architecture
+
+![Architecture overview of a hybrid ETL process using Data Factory][architecture-diagram]
+
+1. Data is sourced from Blob storage into Data Factory.
+2. The Data Factory pipeline invokes a stored procedure to execute an SSIS job hosted on-premises via the Data Factory Integrated Runtime.
+3. The data cleansing jobs are executed to prepare the data for downstream consumption
+4. Once the data cleansing task completes successfully, a copy task is executed to load the clean data into Azure
+5. The clean data is then loaded into tables in the SQL Data Warehouse.
+
+### Components
+
+* [Blob storage][docs-blob-storage] is used to store files and as a source for Data Factory to retrieve data.
+* [SQL Server Integration Services][docs-ssis] contains the on-premises ETL packages used to execute task-specific workloads.
+* [Azure Data Factory][docs-data-factory] is the cloud orchestration engine that takes data from multiple sources and combines, orchestrates, and load the data into a data warehouse.
+* [SQL Data Warehouse][docs-sql-data-warehouse] is used to centralize data in the cloud for easy access using standard ANSI SQL queries.
+
+### Alternatives
+
+Data Factory could invoke data cleansing procedures implemented using other technologies, such as a Databricks notebook, Python script, or SSIS instance running in a virtual machine. [Install paid or licensed custom components on Azure-hosted IR][] may be an alternative to the hybrid approach.
+
+## Considerations
+
+Integrated Runtime (IR) supports two models, self-hosted IR or Azure-hosted IR.  You must first decide between these two options.  Self-hosting is more cost effective but requires more overhead for you to maintain and manage. You can learn more at [Self-hosted IR](https://docs.microsoft.com/en-us/azure/data-factory/concepts-integration-runtime#self-hosted-integration-runtime).  If you need help determining which IR to use you can read more on “[Determine which IR to use](https://docs.microsoft.com/en-us/azure/data-factory/concepts-integration-runtime#determining-which-ir-to-use)”
```

Comment:
For links to docs.microsoft.com, just use the URL path (not including the locale)

e.g., `/azure/data-factory/concepts-integration-runtime#determining-which-ir-to-use`